src/mono/browser/runtime/loader/exit.ts

```diff
@@ -1,7 +1,7 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-import MonoWasmThreads from "consts:monoWasmThreads";
+import WasmEnableThreads from "consts:monoWasmThreads";
```

Comment:
Are we going to rename also the constant?

Reply:
good catch